### ISSUE TITLE
Geo Command: Selecting Navigation causes navigation to start, just to disappear right away

### ISSUE THREAD

**Issue by GoogleCodeExporter** [baluragala/gtalksms#227]
```
The command:

"geo:#address#": Open Maps or Navigation or Street view on specific address.

Fail with Navigation application. The Navigation aplication wont start as 
spected.

Tested on:
Android 2.2.1
Android 2.3.5
HTC Magic
```

Original issue reported on code.google.com by `jdkst...@gmail.com` on 21 Oct 2011 at 10:32

**Comment by GoogleCodeExporter** [comment]
```
Unable to reproduce, please provide an app log.
```

Original comment by `fschm...@gmail.com` on 22 Oct 2011 at 11:07

**Comment by GoogleCodeExporter** [comment]
Original comment by `fschm...@gmail.com` on 22 Oct 2011 at 11:31
- Changed state: **UserFeedBack**

**Comment by GoogleCodeExporter** [comment]
```
Alright, could reproduce that. If you select Navigation it starts just to 
disappear right away. Selecting Map works.
```

Original comment by `fschm...@gmail.com` on 22 Oct 2011 at 11:52
- Changed title: **Geo Command: Selecting Navigation causes navigation to start, just to disappear right away**
- Changed state: **Accepted**

**Comment by GoogleCodeExporter** [comment]
Original comment by `Florent....@gmail.com` on 22 Oct 2011 at 12:53
- Changed state: **Fixed**

**Comment by GoogleCodeExporter** [comment]
```
Thanks for your fantastic job!!

Are any "nigtlhly build" available to test ??
```

Original comment by `jdkst...@gmail.com` on 22 Oct 2011 at 1:33

**Comment by GoogleCodeExporter** [comment]
```
Thanks, no nightly build but I will provide a new beta version in few minutes 
on the download section of the website
```

Original comment by `Florent....@gmail.com` on 22 Oct 2011 at 1:36

**Comment by GoogleCodeExporter** [comment]
```
Version 3.2.3 ready to test
```

Original comment by `Florent....@gmail.com` on 22 Oct 2011 at 1:46

**Comment by GoogleCodeExporter** [comment]
```
Tested and working  !!!

Congratulations for your fantastic job.
```

Original comment by `jdkst...@gmail.com` on 22 Oct 2011 at 3:03

**Comment by GoogleCodeExporter** [comment]
```
Fixed and published with 3.3
```

Original comment by `Florent....@gmail.com` on 11 Nov 2011 at 9:47
- Changed state: **Deployed**
